### ISSUE TITLE
Processes plugin creates strange config

### ISSUE THREAD

**Issue by gytisgreitai** [voxpupuli/puppet-collectd#595]
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: 3.8.7
- Ruby: 1.9.3p484
- Distribution: Ubuntu 14.04
- Module version: 5.1

## How to reproduce (e.g Puppet code you use)
Code taken from README.md:
```
  collectd::plugin::processes::process { 'collectd' : }
  collectd::plugin::processes::processmatch { 'elasticsearch' :
    regex => '.*java.*org.elasticsearch.bootstrap.Elasticsearch'
  }
```
## What are you seeing
It generates the following once run on the server:
```
10-processes.conf:
<LoadPlugin processes>
  Globals false
</LoadPlugin>
```

## What behaviour did you expect instead
I would expect it to contain definitions for elastic and collectd service.

If i use 
   ```
 class { 'collectd::plugin::processes':
      processes => ['process1', 'process2'],
      process_matches => [
        { name => 'process-all', regex => 'process.*' }
      ],
    }
```
everything works as expected

## Output log

> Info: Concat[/etc/collectd/conf.d/processes-config.conf]: Scheduling refresh of Service[collectd]
> Info: Class[Collectd::Config]: Scheduling refresh of Class[Collectd::Service]
> Info: Class[Collectd::Service]: Scheduling refresh of Service[collectd]

**Comment by juniorsysadmin** [comment]
It *looks* like https://github.com/voxpupuli/puppet-collectd/blob/master/examples/plugins/processes.pp is correct, README is wrong.

**Comment by oscarkraemer** [comment]
This seems to be the same issue as #652

**Comment by juniorsysadmin** [comment]
Fixed by #712
